test/services/helpers/services/problems.js

```diff
@@ -0,0 +1,34 @@
+const problemModel = require('../../../../src/services/helpdesk/model');
+
+let createdproblemsIds = [];
+
+const TEST_PROBLEM_PARAMS = {
+	type: 'contactAdmin',
+	_id: '5836bb5664582c35df3bc214',
+	subject: 'Dies ist ein Titel',
+	currentState: 'Dies ist der CurrentState',
+	targetState: 'Dies ist der TargetState',
+	schoolId: '5836bb5664582c35df3bc000',
+};
+const createTestProblem = async (user, problemParameters = TEST_PROBLEM_PARAMS) => {
```

Comment:
can both parameters here being merged? user, problemParameters => params? and directly add userId into params?

Reply:
I would prefer to leave user separate, because this parameter is mostly rewritten. If we will merge both, then all parameters should be set by the caller of the function.

Reply:
just test in the repo that byUser is returning users elements if they exist, it does not return elements of other users which may exist, returns nothing when a user not has elements